collections/_article/godot-xr-update-oct-2024.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Godot XR update - October 2024"
+excerpt: "Looking back at a year of XR development."
+categories: ["progress-report"]
+author: Bastiaan Olij
+image: /storage/blog/covers/october-2024-xr-progress-update.webp
+image_caption_description: "Untitled penguin VR project by Zi Ye"
+date: 2024-10-01 11:00:00
+---
+
+It has been a very busy year on the XR front and seeing we haven’t given a proper update for well over a year, it’s time to look back at what has happened since our last update post.
+We touched on some of these in the Godot 4.3 release blog but we’re doing a bit of a deep dive here.
+
+We’re planning on more XR focused posts in the near future, we recently [announced the XR editor](https://godotengine.org/article/godot-editor-horizon-store-early-access-release/) but we have more news on the way.
```

Comment:
This sentence is a little ambiguous about whether we're going to do "more posts about XR" (the intended meaning) or "posts that are more focused/detailed".

How about:

```suggestion
We’re planning on publishing more articles about XR in the near future. We recently [announced the XR editor](https://godotengine.org/article/godot-editor-horizon-store-early-access-release/) but we have more news on the way!
```

(Snuck in some punctuation notes too :-))